### DISCUSSION TITLE
Azure Data Studio

### DISCUSSION THREAD

**Discussion by dkmix** [microsoft/azuredatastudio#24068] [Q&A]
I'm getting the error below when running schema compare with Azure SQL database. I can connect to the database fine when just using the database connection. error only when trying to run compare. it seems that the schema compare not using the right token?

"Schema Compare failed: Login failed for user token-identified principal

![Screenshot 2023-08-03 141335](https://github.com/microsoft/azuredatastudio/assets/56991476/ea387fb3-0995-4022-8874-02ee7c03b285)

**Comment by erinstellato-ms** [comment]
Hi @dkmix - could you please log this as an issue (https://github.com/microsoft/azuredatastudio/issues) and include the ADS version and exact steps to recreate so that we can investigate?  Thank you!
